src/blocks/ImageTextBlock/index.tsx

```diff
@@ -142,6 +170,14 @@ const ImageLink = styled('a')(
   }),
 )
 
+const ImageVideo = styled(DeferredVideo)({
+  width: '100%',
+  objectFit: 'cover',
```

Comment:
This guy is a bit dangerous because the browser support is fairly poor still. Just double checking it looks *decent* also without this property. see https://caniuse.com/#search=object-fit

Reply:
Yes, I'm aware of that! I figured since it's already being used (such as in `HomeHeroBlock`) I could just continue with that. And I guess if it's looking decent even without it, it's not a problem that it's not working in some old browsers? 😄

Reply:
Tots :)